juriscraper/pacer/docket_report.py

```diff
@@ -1695,7 +1695,7 @@ def _get_case_name(self):
         return clean_string(harmonize(case_name))
 
     def _parse_docket_number(
-        self,
+        self, title_text: Optional[str] = None
```

Comment:
Is there a reason to add `title_text` as a parameter to this method? Considered is not used in the method.